packages/schemas/src/types/log/interaction.ts

```diff
@@ -68,8 +71,8 @@ export enum Action {
  */
 export type LogKey =
   | `${Prefix}.${Action.Create | Action.End}`
-  | `${Prefix}.${InteractionEvent}.${Action.Update | Action.Submit}`
-  | `${Prefix}.${InteractionEvent}.${Field.Profile}.${Action.Update}`
+  | `${Prefix}.${InteractionEvent}.${Action.Update | Action.Submit | Action.Delete}`
```

Comment:
did we update admin console display accordingly?

Reply:
Title? Updated